telegram/_bot.py

```diff
@@ -828,8 +833,23 @@ async def send_message(
 
                 .. versionchanged:: 20.0
                     |sequenceargs|
+            link_preview_options (:obj:`LinkPreviewOptions`, optional): Link preview generation
+                options for the message. Mutually exclusive with
+                :paramref:`disable_web_page_preview`.
+
+                .. versionadded:: NEXT.VERSION
+
             disable_web_page_preview (:obj:`bool`, optional): Disables link previews for links in
-                this message.
+                this message. Mutually exclusive with :paramref:`link_preview_options`.
+
+                .. versionchanged:: NEXT.VERSION
+                    Bot API 7.0 introduced :paramref:`link_preview_options` replacing this
+                    argument. PTB will automatically convert this argument to that one, but
+                    you should update your code to use the new argument.
```

Comment:
I should convert only if I want to use the advanced features, right? Since we're planning to keep dwpp for convenience?

Reply:
correct, but don't we want to recommend to use that new argument anyway?